pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "apistar"
+version = "0.5.26"
+description = "Blazingly fast & beautifully expressive Web APIs"
+authors = ["Tom Christie <tom@tomchristie.com>"]
+#license = "BSD"
+readme = "README.md"
```

Comment:
does this do the proper stuff to say to PyPI that its a `text/markdown` type?

Reply:
the poetry project itself uses README.md which looks good on pypi : https://pypi.org/project/poetry/